### DISCUSSION TITLE
🎚️⬆️ Boosts Summary Screen - Full Island

### DISCUSSION THREAD

**Discussion by GoliathRulz** [sunflower-land/sunflower-land#2078] [Ideas]
Need to have an in-game screen that summarizes all the current active boosts for the island, which would include the bumpkin skills as well as placed NFTs.  Currently there is no way to see an overall summary in the game and this need to be handled out of the game.  For players there should be a status screen that shows this data in-game.

**Comment by IvanSkazkin** [comment]
nice idea
